libs/features/unused-guid-search/src/unused-guid-search.component.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { of, throwError } from 'rxjs';
+import { UnusedGuidSearchComponent } from './unused-guid-search.component';
+import { MysqlQueryService } from '@keira/shared/db-layer';
+
+describe('UnusedGuidSearchComponent', () => {
+  let fixture: ComponentFixture<UnusedGuidSearchComponent>;
+  let queryService: MysqlQueryService;
```

Comment:
let's avoid shared variables across tests and instead initialize and return these from the `setup` function (like you already do with the `component` object)